app.js

```diff
@@ -1,8 +1,140 @@
+// Return the digit in the ones place of an integer
+function getOnesDigit(num) {
+  return num % 10;
+}
+
+// Return the digit in the tens place of an integer
+function getTensDigit(num) {
+  num %= 100;
+  return Math.floor(num / 10);
+}
+
+// Return the digit in the hundreds place of an integer
+function getHundredsDigit(num) {
+  num %= 1000;
+  return Math.floor(num / 100);
+}
+
+// Return the digit in the thousands place of an integer
+function getThousandsDigit(num) {
+  return Math.floor(num / 1000);
+}
+
+// Convert a digit located in the ones place of an integer
+// to the equivalent roman numeral
+function convertOnesDigitToRoman(digit) {
+  // Convert 9 to IX
+  if (digit === 9) return 'IX';
+
+  // Convert 4 to IV
```

Comment:
Happy to see more of these inside-the-function comments. Reading (and understanding) your code this time was as smooth as possible--nice!

Reply:
Not that it was hard to understand last time per se, just that it's even smoother/faster with these comments